client/experimentalclient/client.go

```diff
@@ -1,4 +1,4 @@
-package experimental
+package experimentalclient
```

Comment:
I have figured out why the CI failed.
I can reproduce it by importing this package.
The init() has unexpected impact.